builders/integrations/indexers/covalent.md

```diff
@@ -46,70 +56,358 @@ If you're already familiar with Covalent and ready to dive in, you simply need t
+
+    === "Parameters"
+
+        - `chainName` *string* - e.g. `moonbeam-mainnet`, `moonbeam-moonriver`, or `moonbeam-moonbase-alpha` 
+        - `walletAddress` *string* - the address you wish to query
+        - `timeBucket` *integer* - The 0-indexed 15-minute time bucket. E.g. 8 9 Oct 2024 01:49 GMT = 1728420540 (Unix time). 1728420540/900 = 1920467 timeBucket.
+
+    === "Example Request"
+
+        ```bash
+        --8<-- 'code/builders/integrations/indexers/covalent/get-bulk-time/request.sh'
+        ```
+
+    === "Example Response"
+
+        ```bash
+        --8<-- 'code/builders/integrations/indexers/covalent/get-bulk-time/response.sh'
+        ```
+
+??? function "Get all transactions in a block by page (v3)"
+
+    [Get all transactions in a block by page (v3)](https://goldrush.mintlify.app/docs/api/transactions/get-all-transactions-in-a-block-by-page){target=\_blank} is used to fetch all transactions including their decoded log events in a block and further flag interesting wallets or transactions.
+
+    === "Parameters"
+
+        - `chainName` *string* - e.g. `moonbeam-mainnet`, `moonbeam-moonriver`, or `moonbeam-moonbase-alpha` 
+        - `blockHeight` *integer* - the request block height. Also accepts the `latest` keyword
+        - `page` *integer* - the requested page, 0-indexed.
+
+    === "Example Request"
+
+        ```bash
+        --8<-- 'code/builders/integrations/indexers/covalent/get-all-txns-in-block/request.sh'
+        ```
+
+    === "Example Response"
+
+        ```bash
+        --8<-- 'code/builders/integrations/indexers/covalent/get-all-txns-in-block/response.sh'
+        ```
+
+??? function "Get all transactions in a block (v3)"
+
+    [Get all transactions in a block (v3)](https://goldrush.mintlify.app/docs/api/transactions/get-all-transactions-in-a-block){target=\_blank} is used to used to fetch all transactions including their decoded log events in a block and further flag interesting wallets or transactions. It takes a blockhash as a parameter and it does not accept a page parameter.
+
+    === "Parameters"
+
+        - `chainName` *string* - e.g. `moonbeam-mainnet`, `moonbeam-moonriver`, or `moonbeam-moonbase-alpha` 
+        - `blockHash` *integer* - the request block hash
+
+    === "Example Request"
+
+        ```bash
+        --8<-- 'code/builders/integrations/indexers/covalent/all-txns-by-block-hash/request.sh'
+        ```
+
+    === "Example Response"
+
+        ```bash
+        --8<-- 'code/builders/integrations/indexers/covalent/all-txns-by-block-hash/response.sh'
+        ```
+
+??? function "Get ERC-20 Token Transfers for Address"
+
+    [Get ERC-20 Token Transfers for Address](https://goldrush.mintlify.app/docs/api/transactions/get-erc20-token-transfers-for-address){target=\_blank} is used to fetch the transfer-in and transfer-out of a token along with historical prices from an address, when provided both a wallet address and an ERC-20 token contract address
```

Comment:
I think they moved this one on you since you submitted it. It looks like it's part of Balances now instead of Transfer. This fixes the link, but please move it up to the Balances section. Thanks!